src/features/auth/ui/SignUpLink.tsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router';
+
+export const SignUpLink = () => {
+  return (
+    <div className="flex justify-center items-center mt-6">
+      <p className="text-lg">계정이 없으신가요?</p>
+      <Link
+        to="/signup"
+        className="ml-2 text-2xl font-semibold text-[#329A6A] hover:underline"
```

Comment:
```suggestion
        className="ml-2 text-2xl font-semibold text-(--color-primary) hover:underline"
```
와 같이 상수선언해서 적용하는것도 좋아보입니다! 나중에 theme설정할때도 유용합니다😀

Reply:
@zzzRYT 제가 요즘 고민중인 게 있는데요! 마침 리뷰 남겨주셔서 떠올릴 수 있었습니다. 우선 리뷰 정말 감사합니다! 이번 프로젝트는 디자이너 분들이 작업해주시는 피그마를 보고 색상을 이렇게 대입해서 넣고 있어요.(미니때는 자주 사용하는 색상 2개만 팔레트에 넣어서 사용했었고요!) 서비스 관점에서 일관된 팔레트 같은 게 있으면 좋겠다는 생각은 하고 있지만, 그게 뭔가 저희 팀의 작업 속도를 늦출 것 같다는 생각도 들어서요. 재진님은 혹시 팔레트를 정하고 가는 게 낫다고 생각하시나요? 어디서부터 시작해야할지 감이 안잡히네요... 😣

Reply:
@electrohyun 제가 코멘트를 너무 늦게 봤네요, 팔레트작업, `디자인 토큰`이라고 부르더라구요. 제 생각에는 빠르게 정의하면 정의할 수록 좋다고 생각합니다. 추후에 디자인색상이 바뀌어도 그 부분만 바꾸면 그만이니깐요!🙃